archives/archives_test.go

```diff
@@ -322,53 +322,23 @@ func TestArchiveOrgMessages(t *testing.T) {
 
 		assertCount(t, db, 4, `SELECT count(*) from msgs_broadcast WHERE org_id = $1`, 2)
 
-		created, deleted, err := ArchiveOrg(ctx, now, config, db, s3Client, orgs[1], MessageType)
+		dailiesCreated, dailiesFailed, monthliesCreated, monthliesFailed, deleted, err := ArchiveOrg(ctx, now, config, db, s3Client, orgs[1], MessageType)
 		assert.NoError(t, err)
 
-		assert.Equal(t, 63, len(created))
-		assert.Equal(t, time.Date(2017, 8, 10, 0, 0, 0, 0, time.UTC), created[0].StartDate)
-		assert.Equal(t, DayPeriod, created[0].Period)
-		assert.Equal(t, 0, created[0].RecordCount)
-		assert.Equal(t, int64(23), created[0].Size)
-		assert.Equal(t, "f0d79988b7772c003d04a28bd7417a62", created[0].Hash)
-
-		assert.Equal(t, time.Date(2017, 8, 11, 0, 0, 0, 0, time.UTC), created[1].StartDate)
-		assert.Equal(t, DayPeriod, created[1].Period)
-		assert.Equal(t, 0, created[1].RecordCount)
-		assert.Equal(t, int64(23), created[1].Size)
-		assert.Equal(t, "f0d79988b7772c003d04a28bd7417a62", created[1].Hash)
-
-		assert.Equal(t, time.Date(2017, 8, 12, 0, 0, 0, 0, time.UTC), created[2].StartDate)
-		assert.Equal(t, DayPeriod, created[2].Period)
-		assert.Equal(t, 3, created[2].RecordCount)
-		assert.Equal(t, int64(528), created[2].Size)
-		assert.Equal(t, "b3bf00bf1234ea47f14ffd0171a8ead0", created[2].Hash)
-
-		assert.Equal(t, time.Date(2017, 8, 13, 0, 0, 0, 0, time.UTC), created[3].StartDate)
-		assert.Equal(t, DayPeriod, created[3].Period)
-		assert.Equal(t, 1, created[3].RecordCount)
-		assert.Equal(t, int64(312), created[3].Size)
-		assert.Equal(t, "32e61b1431217b59fca0170f637d78a3", created[3].Hash)
-
-		assert.Equal(t, time.Date(2017, 10, 10, 0, 0, 0, 0, time.UTC), created[60].StartDate)
-		assert.Equal(t, DayPeriod, created[60].Period)
-		assert.Equal(t, 0, created[60].RecordCount)
-		assert.Equal(t, int64(23), created[60].Size)
-		assert.Equal(t, "f0d79988b7772c003d04a28bd7417a62", created[60].Hash)
-
-		assert.Equal(t, time.Date(2017, 8, 1, 0, 0, 0, 0, time.UTC), created[61].StartDate)
-		assert.Equal(t, MonthPeriod, created[61].Period)
-		assert.Equal(t, 4, created[61].RecordCount)
-		assert.Equal(t, int64(553), created[61].Size)
-		assert.Equal(t, "156e45e29b6587cb85ccf75e03800b00", created[61].Hash)
-
-		assert.Equal(t, time.Date(2017, 9, 1, 0, 0, 0, 0, time.UTC), created[62].StartDate)
-		assert.Equal(t, MonthPeriod, created[62].Period)
-		assert.Equal(t, 0, created[62].RecordCount)
-		assert.Equal(t, int64(23), created[62].Size)
-		assert.Equal(t, "f0d79988b7772c003d04a28bd7417a62", created[62].Hash)
-
-		// no rollup for october since that had one invalid daily archive
```

Comment:
despite what this comment claims, the test data doesn't have any failing messages